### DISCUSSION TITLE
Do I need to install sound files?

### DISCUSSION THREAD

**Discussion by hereww** [War3Evo/War3Source-EVO#28] [Q&A] [answered]

**Comment by JustZerooo** [comment]
You should do it yes

**Reply by hereww** [reply]
= =！！！！！6  year！！！！！！！

**Reply by hereww** [reply]
It's been six years of waiting, and although it's surprising, if I do a Chinese localization, I would like to set up a CSGO war3 server. If there are any issues, can I find you first and seek a solution?

**Reply by War3Evo** [reply, marked as answer]
# Project Update and Status

If the required sound files are not installed, the user's console will be flooded with error messages. Advanced users may notice this issue, and in TF2, it can potentially cause performance lag.

**Note:** You are free to use any replacement sounds you prefer; there’s no need to use the original ones.

## CSGO Version Status

At the time I stopped updating this repository, the CSGO version was still a work in progress. While the menus and some races functioned, the graphical elements were incomplete due to the different way CSGO handles them compared to TF2.

I initially took over this project from Ownz, who worked on it for CS: Source, but I didn't have the benefit of learning the CS engine. I attempted to create functional variants, which do work, but they’re not as polished as TF2.

Additionally, my current computer can barely handle CSGO, making debugging a chore. To properly finish the CSGO version, I would need a more powerful machine.

### Branch Information

- **Master Branch**: This is the polished version.
- **Develop Branch**: I was working hard to implement translations here. For it to work in your language, you would need to handle all the translations yourself, as the master branch only supports a single language. However, if you'd like to contribute, you can help finish the develop branch to enable translations for everyone.

## Progress

It’s frustrating when people ask about a fully polished CSGO version—it’s only about **50% complete**. Several races still require graphical assets and sounds. While the engine is functional, the races are the key part of the game.

## Customization

If you learn **SourcePawn**, you should be able to modify the code to suit your specific needs.

## Future Plans

At some point, I plan to archive the CSGO version and move on. It was never my goal to complete it fully; I only worked on it out of curiosity and because a few friends were interested at the time. That was quite some time ago.
